docs/rulebooks.rst

```diff
@@ -138,6 +138,31 @@ To avoid name conflicts the source data structure can use nested keys.
 If any source terminates, it shuts down the whole engine. All events from other sources may be lost.
 
 
+Using vaulted strings
+--------------------------
+
+Sensitive data referenced by a rulebook should be encrypted by ansible-vault cli. The vaulted strings
```

Comment:
I think we should add a warning message [like ansible does](https://docs.ansible.com/ansible/latest/vault_guide/vault.html) to make clear that once the variable is decrypted is responsibility of the user to not expose it (like for example printing it in the debug action)